suite-native/atoms/package.json

```diff
@@ -13,6 +13,7 @@
     "dependencies": {
         "@gorhom/bottom-sheet": "5.1.5",
         "@mobily/ts-belt": "^3.13.1",
+        "@react-navigation/native": "6.1.18",
```

Comment:
I would advise against this and send navigational (even goBack) functions as callbacks to atoms